### PR TITLE
Week 7 Tasks - Juay Hee

### PR DESCRIPTION
**Pull request by juayhee**
1. Update README with Juay Hee's information
2. Update build status badge link to use our repository instead of AB3
3. Acknowledge original source of code
4. Add skeletal PPP for Juay Hee

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -19,15 +19,16 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Project Advisor
 
-### Jane Doe
+### Tan Juay Hee
 
 <img src="images/johndoe.png" width="200px">
```
Comment (AAlghrairy) [inline docs/AboutUs.md:24]: Should also upload own photo (or a placeholder) and rename the photo to your username (in lowercase), as the tracker also checks for this.

**Review by AAlghrairy — COMMENTED** [review]
Looks Good!

Just remember to upload a photo (or placeholder) and rename the appropriate line `AboutUs.md`.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
+  * as an organisation tool for teams and organisations
+
+## Features
+
+- Full contact management
+  - List all your contacts
+  - Add, delete and edit them
+  - Add notes to contacts
+
+- Tags
```
[the remark this answers is not in the document]
Comment (juayhee) [reply README.md:21]: I see where you are coming from, but then everything would be under contact management, basically. For now, what about we remove line 16?

[the remark this answers is not in the document]
Comment (juayhee) [reply README.md:21]: And then when we get the visualisation thing going we can split it into Contact Management and Strategic Planning/whatever

[the remark this answers is not in the document]
Comment (simjunheng) [reply README.md:21]: Yeap, we have other sections e.g. Strategic Planning and Scheduler (name not confirmed) in the later part of the project hence I don't think everything will be under full contact management

[the remark this answers is not in the document]
Comment (simjunheng) [reply README.md:21]: At least for now, it would seem that everything is under full contact management

[the remark this answers is not in the document]
Comment (simjunheng) [reply README.md:21]: I think you can go ahead and remove line 16 if you want but we will likely add it back at a later date when we start working on scheduling etc

File: docs/AboutUs.md (hunk 1 above)
Comment (Rye-Catcher) [reply docs/AboutUs.md:24]: agree

**Pull request merged**